scripts/mbedtls_dev/ecp.py

```diff
@@ -144,6 +144,86 @@ def result(self) -> List[str]:
     def is_valid(self) -> bool:
         return True
 
+class EcpPp384R1Raw(bignum_common.ModOperationCommon,
+                    EcpTarget):
+    """Test cases for ecp quasi_reduction modulo p384."""
+    test_function = "ecp_mod_p384_raw"
+    test_name = "ecp_mod_p384_raw"
+    input_style = "arch_split"
+    arity = 1
+
+    moduli = [("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+               "fffffeffffffff0000000000000000ffffffff")
+             ] # type: List[str]
+
+    input_values = [
```

Comment:
Test cases for the second round of carry reduction (when the first round generates a carry) is also needed.